src/shogun/base/SGObject.cpp

```diff
@@ -1059,6 +1070,15 @@ CSGObject* CSGObject::create_empty() const
 	return object;
 }
 
+void CSGObject::initialise_auto_params()
```

Comment:
could we name it "init" to avoid clashes between brits and US? :)

Reply:
haha sure!